backend/src/main/java/com/exadel/carinsurance/model/UserEntity.java

```diff
@@ -34,6 +35,7 @@ public class UserEntity implements UserDetails {
   private String email;
 
   @Column( name = "password" )
+  @JsonIgnore

```

Comment:
it is better to have separate models for DAO and Controller layer. That way it is harder to expose anything unnecessary during development